src/routes.py

```diff
@@ -9,5 +12,12 @@
 @bp.route("/")
 @bp.route("/index")
 def index() -> str:  # pylint: disable=unused-variable
-    """Use this view function to check whether Flask is installed properly."""
-    return "Hello, wordl!"
+    """Use this functon to render index page's template."""
+    return render_template("index.html")
```

Comment:
Let's keep our `/index` page unchanged because it's not in the scope of current task. We will definitely change it later, but it'll be done as a part of other task.

As they say:
> Joe wasn't part of our deal.
![image](https://user-images.githubusercontent.com/35929293/211121482-8da93abd-8d6f-43aa-a9f8-c20751a53779.png)